airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -84,6 +85,32 @@ def get_es_kwargs_from_config() -> dict[str, Any]:
     return kwargs_dict
 
 
+def _ensure_ti(ti: TaskInstanceKey | TaskInstance, session) -> TaskInstance:
```

Comment:
@pankajkoti this is duplicated from FileTaskHandler

if we import from core, then we have to bump major

also, i don't like to create public interface when we can avoid it

now it's a little code duplication, but it's private so we can remove it at will.

@uranusjr might have comment. this is the bit of code that originally i put as class method under TI key

Reply:
Okay, thanks for the push 🙏🏽. I din't test the ES remote logging yet, will test it soon and mark the PR ready for review once tested